### PR TITLE
Bugfix multi input field

### PR DESCRIPTION
**Pull request by srh-sloan**
### Change description
Fixes the import and export of components of type MultiInputField (add another).

Updated the DB schema:
#### Component
- added `children` (stores the child components of multi input field)
- added `schema` (stores some info about field length etc on number fields)
- added `MultiInputField` as a valid component type in the enum
#### Page
- Added the `options` column to store config used by the `RepeatingFieldPageController`

Updated unit tests accordingly

- [x] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [x] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

File: app/db/migrations/versions/~2024_10_16_1347-da30746cec39_.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+revision = 'da30746cec39'
+down_revision = 'ca61d3b746f6'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+        
+    op.execute("ALTER TYPE componenttype ADD VALUE 'MULTI_INPUT_FIELD';")
```
Comment (nuwan-samarasinghe) [inline app/db/migrations/versions/~2024_10_16_1347-da30746cec39_.py:23]: In multiinputfield we have to have RepeatingFieldPageController as the controller & 
`"options": {
                "summaryDisplayMode": {
                    "samePage": true,
                    "separatePage": false,
                    "hideRowTitles": false
                },
                "customText": {
                    "samePageTitle": "Your secured match funding"
                }
            },`

options to show up the table is it already handling ?

Comment (srh-sloan) [reply app/db/migrations/versions/~2024_10_16_1347-da30746cec39_.py:23]: Yes it's added in this same pr :)

File: app/db/migrations/versions/~2024_10_16_1347-da30746cec39_.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    with op.batch_alter_table('component', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('children', postgresql.JSON(none_as_null=True, astext_type=sa.Text()), nullable=True))
+        batch_op.add_column(sa.Column('schema', postgresql.JSON(astext_type=sa.Text()), nullable=True))
+    with op.batch_alter_table('page', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('options', postgresql.JSON(none_as_null=True, astext_type=sa.Text()), nullable=True))
```
Comment (srh-sloan) [inline app/db/migrations/versions/~2024_10_16_1347-da30746cec39_.py:28]: Here @nuwan-samarasinghe

**Review by nuwan-samarasinghe — APPROVED** [review]
LGTM

**Pull request merged**